### DISCUSSION TITLE
video is downloaded in very small parts

### DISCUSSION THREAD

**Discussion by JuanAliagaPerez** [subhra74/xdm#674] [Q&A]
Hi, I've been using XDM for about a year now and have never had a problem downloading videos before; A few weeks ago the "download file" button stopped appearing, I changed the minimum capture size and it was shown again, but many videos of 31 kbs in size or less began to appear; When downloading the individual videos cannot be played (each "video" is only one frame of the total video).

Does anyone know how I can download the full video?

Thank you!

**Comment by L420624me** [comment]
yep,I also have this kind of problem, please ask you to solve it?
